### PR TITLE
Feature/us12827 404 page

### PR DESCRIPTION
**Pull request by zarahzachz**
Add 404 page. Per [documentation](https://www.netlify.com/docs/redirects/#custom-404), a custom 404 should just work in Netlify.

No corresponding PR.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -5,7 +5,6 @@ baseurl: "" # the subpath of your site, e.g. /blog
 url: "https://www.crossroads.net" # the base hostname & protocol for your site, e.g. http://example.com
 
 # disable_hubspot: true
-permalink: pretty
```
Comment (tcmacdonald) [inline _config.yml]: @zarahzachz Can you please revert this change?

**Comment by zarahzachz** [comment]
@tcmacdonald I added the pretty links back. Do your magic.
